### PR TITLE
Solve issue where preload links had comma's between them.

### PR DESCRIPTION
**Pull request by paales**

### CODE REVIEW

File: packages/building-webpack/modern-web-webpack-plugin.js [hunk 1]
```diff
@@ -211,7 +211,7 @@ module.exports = class LegacyBrowserWebpackPlugin {
           const preloadLinks = this.entries.modern.map(
             src => `<link rel="preload" href="${src}" as="script">`,
           );
-          data.html = data.html.replace('</head>', `${preloadLinks}</head>`);
+          data.html = data.html.replace('</head>', `${preloadLinks.join("\n")}</head>`);
```
Comment (LarsDenBakker) [inline packages/building-webpack/modern-web-webpack-plugin.js]:
```suggestion
          data.html = data.html.replace('</head>', `${preloadLinks.join('')}</head>`);
```
save a few bytes :)

**Comment by LarsDenBakker** [comment]
Oops, thanks for this! Looks like you need to run prettier with `npm run format`

**Comment by paales** [comment]
@LarsDenBakker I haven't got this repo installed locally from source, so I can't run `npm run format`, is that something you can do?

**Comment by LarsDenBakker** [comment]
Looks like the change to '' fixed the formatting too. Thanks for your contribution!

**Pull request merged**